File: patient/patidata/src/main/java/com/pecenio/patims/model/Patient.java
package com.pecenio.patims.model;

import lombok.Data;
import java.util.Date;

@Data
public class Patient {
	private int id;
	private Long userId;
	private String username;
	private String password;
	private String role;
	private String firstName;
	private String lastName;
	private String phoneNumber;
	private String dateOfBirth;
	private int age;
	private String address;
	private String emergencyContact;
	private Date lastUpdated;
	private Date created;
}

File: PatientApp/app/src/main/java/com/example/patientapp/data/model/Patient.kt
package com.example.patientapp.data.model

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class Patient(
    val id: Int = 0,
    val userId: Long? = null,
    val username: String = "",
    val password: String = "",
    val role: String = "PATIENT",
    val firstName: String = "",
    val lastName: String = "",
    val phoneNumber: String = "",
    val dateOfBirth: String = "",
    val age: Int = 0,
    val address: String = "",
    val emergencyContact: String = ""
) : Parcelable
